ruamel_yaml/meta.yaml

```diff
@@ -29,4 +31,6 @@ test:
 about:
   home: https://bitbucket.org/ruamel/yaml
   license: MIT
+  license_family: MIT
   license_file: LICENSE
+  summary: A patched copy of ruamel.yaml.
```

Comment:
Figured there should be some explanation of what this is as it has caused confusion before.

xref: https://github.com/conda-forge/ruamel_yaml-feedstock/issues/1
xref: https://github.com/conda-forge/ruamel.yaml-feedstock/issues/7